pkg/remote/oci/resolver_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package oci
+package oci_test
```

Comment:
Just curious, why did this need to change?

Reply:
This came out of the use of the constants for the key names. I figured it would be better practice to put the test in a different package space so that we were testing the interface rather than the internals.

Reply:
Ah ok, makes sense.